packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -1902,6 +1902,77 @@ describe('cluster', () => {
       DBClusterIdentifier: clusterIdentifier,
     });
   });
+
+  test('allows accessing instanceIdentifiers', () => {
+    // GIVEN
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    const cluster = new DatabaseCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.aurora({
+        version: AuroraEngineVersion.VER_1_22_2,
+      }),
+      credentials: {
+        username: 'admin',
+      },
+      instanceProps: {
+        vpc,
+      },
+      instances: 1,
+    });
+
+    // THEN
+    expect(cluster.instanceIdentifiers).toHaveLength(1);
+    expect(stack.resolve(cluster.instanceIdentifiers[0])).toEqual({
+      Ref: 'DatabaseInstance1844F58FD',
+    });
+  });
+
+  test('allows accessing instanceEndpoints', () => {
+    // GIVEN
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    const cluster = new DatabaseCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.aurora({
+        version: AuroraEngineVersion.VER_1_22_2,
+      }),
+      credentials: {
+        username: 'admin',
+      },
+      instanceProps: {
+        vpc,
+      },
+      instances: 1,
+    });
+
+    // THEN
+    expect(cluster.instanceEndpoints).toHaveLength(1);
+    expect(stack.resolve(cluster.instanceEndpoints[0])).toEqual({
+      hostname: {
+        'Fn::GetAtt': ['DatabaseInstance1844F58FD', 'Endpoint.Address'],
+      },
+      port: {
+        'Fn::GetAtt': ['DatabaseB269D8BB', 'Endpoint.Port'],
+      },
+      socketAddress: {
+        'Fn::Join': [
+          '',
+          [
+            {
+              'Fn::GetAtt': ['DatabaseInstance1844F58FD', 'Endpoint.Address'],
+            },
+            ':',
+            {
+              'Fn::GetAtt': ['DatabaseB269D8BB', 'Endpoint.Port'],
+            },
+          ],
+        ],
+      },
+    });
+  });
```

Comment:
This is such a small change, I'm not sure it warrants an entirely separate unit test.

How about adding a new assertion to existing tests?

Reply:
I now included the expectations to an existing unit test that was creating a cluster with 1 db instance.

But I do feel like the existing unit tests have been pretty specific on testing just a certain functionality, so it didn't fit too naturally into any existing test in my opinion.